### PR TITLE
Set fixtures plugin as a library

### PR DESCRIPTION
**Pull request by loic425**
https://github.com/symfony/recipes-contrib/pull/871#discussion_r391319285

### CODE REVIEW

File: src/Monofony/Plugin/FixturesPlugin/composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "monofony/fixtures-plugin",
```
Comment (lchrusciel) [inline src/Monofony/Plugin/FixturesPlugin/composer.json]: I would consider keeping the Symfony naming convention and name it as a bundle. 

Also, can you give me a brief insight, what is added in this library?

Comment (loic425) [reply src/Monofony/Plugin/FixturesPlugin/composer.json]: @lchrusciel Hi :)
- This package contains AbstractResourceFixture and AbstractExampleFactory.
- It adds sylius/fixtures-bundle as dependency
- It copies fixtures and example factory services for admin user and app user on recipe.

Comment (loic425) [reply src/Monofony/Plugin/FixturesPlugin/composer.json]: I don't know what to do... I could copy that into the CoreBundle but it adds a dependency to all Monofony projects and maybe a lot of users don't need that or use their own package.

Comment (lchrusciel) [reply src/Monofony/Plugin/FixturesPlugin/composer.json]: sure, I was just curious. Maybe we should improve these packages in the long term, to provide these classes out-of-the-box, but it is another topic ;)

Comment (loic425) [reply src/Monofony/Plugin/FixturesPlugin/composer.json]: @lchrusciel Could you help me on  https://github.com/symfony/recipes-contrib/pull/871#discussion_r391319285 to talk with @nyholm and trying to fit the 
requirements to be approved?

Comment (loic425) [reply src/Monofony/Plugin/FixturesPlugin/composer.json]: I could give you a new architecture brief of course.

Comment (loic425) [reply src/Monofony/Plugin/FixturesPlugin/composer.json]: @lchrusciel What do you purpose for the naming, cause it's not a bundle anymore?

Comment (lchrusciel) [reply src/Monofony/Plugin/FixturesPlugin/composer.json]: Hey Loic! I would still vote to make it a bundle and name it MonofonyFixturesBundle. I'm considering adding it straight to our FixturesBundle. But we would need to test these classes and add ResourceBundle as dev dependency. 

I cannot promise anything, but maybe I will be able to jump into recipe conversation this week, to be up-to-date with their requirements

Comment (loic425) [reply src/Monofony/Plugin/FixturesPlugin/composer.json]: @lchrusciel Thanks a lot!
Maybe can I help Sylius to add these services on FixturesBundle.

**Pull request closed without merging**